packages/sanity/src/tasks/plugin/TasksFooterOpenTasks.tsx

```diff
@@ -0,0 +1,45 @@
+import {useCallback, useMemo} from 'react'
+
+import {Button} from '../../ui-components'
+import {useTasks, useTasksEnabled} from '../src'
+
+/**
+ * Button that shows how many pending tasks are assigned to the current document.
+ * Clicking it will open the task sidebar, showing the open tasks related to the document.
+ *
+ * todo: just show the tab with tasks related to the document
+ * @internal
+ */
+export function TasksFooterOpenTasks() {
+  const {data, activeDocument, toggleOpen, isOpen} = useTasks()
+  const {enabled} = useTasksEnabled()
+
+  const pendingTasks = useMemo(
+    () =>
+      data.filter((item) => {
+        return item.target?.document._ref === activeDocument?.documentId && item.status === 'open'
+      }),
+    [activeDocument, data],
+  )
+
+  const handleOnClick = useCallback(() => {
+    if (isOpen) {
+      return
+    }
+    toggleOpen()
+  }, [isOpen, toggleOpen])
+
+  if (pendingTasks.length === 0 || !enabled) return null
+
+  const pluralizedTask = `task${pendingTasks.length > 1 ? 's' : ''}`
```

Comment:
Would it be possible to use i18next for these strings? It has built in [support for pluralisation](https://www.i18next.com/translation-function/plurals).

Reply:
We don't have translations yet in `tasks` will be added as a quick follow after release.